documentation/deliverables/individual_deliverables/open-source-group-kickoff.md

```diff
@@ -0,0 +1,148 @@
+
+### Developer Tools Setup  
+
+Provide technical instructions for setting up developer tools and environments. This can encompass version control systems, code review tools, and any collaborative platforms for coding and testing.
+
+
+
+### Definition of done
+
+- [ ] Onboarding experience ready for the public this includes:
+    - [ ] onboarding to the selected chat tool, Slack
+    - [ ] onboarding to Github
+    - [ ] materials such as a deck or another format for getting started with the project
+    - [ ] onboarding guidance for developer tools and environments
+- [ ] The selected video conference tool, Zoom, is procured and ready for use for webinars, office hours, or other open source events. 
+
+### Proposed metrics for measuring goals/value/definition of done
+
+- [ ] Number of users onboarded to the open source community
+- [ ] Time to onboard to the open source community
+
+
+## Planning
+
+### Assumptions & dependencies
+<!-- Required -->
+
+What capabilities / milestones do we expect to be in place at the beginning of work
+on this milestone?
+
+- Procurement of the tools should be in place as the procurement process can delay delivery. Zoom, Slack, and Github are already licensed on the project. 
+
+Are there any notable capabilities / milestones we do NOT expect to be in place at the
+beginning of work on this milestone?
+
+
+
+### Open questions
+<!-- Optional -->
+
+- [ ] Is it important to have a public wiki available and ready in this deliverable?
+- [ ] Is it important to have a public API documentation ready for consumption in this deliverable? 
+
+### Not doing
+<!-- Optional -->
+
+The following work will *not* be completed as part of this milestone:
+- tools that are not stated as required (Google Groups, public wiki, etc) 
+
+## Integrations
+
+### Translations
+<!-- Required -->
+
+
+### Services going into PROD for the first time
+<!-- Required -->
+
+*This can include services going into PROD behind a feature flag that is not turned on.*
+
+Yes the following tools will be deployed:
+
+- Zoom for video conferencing. Only internal teams will have a Zoom license, but the public will join Zoom   
+
+
+### Services being integrated in PROD for the first time
+<!-- Required -->
+
+*Are there multiple services that are being connected for the first time in PROD?*
+
+Yes, we expect there to be some integrations between the following tools in production:
+
+1. **Chat + Ticket tracking:** Option to receive updates on key tickets in chat
+2. **Wiki + Chat:** Option to receive updates on key document changes in chat
+3. **Document Sharing + Chat:** Option to show previews of documents shared in chat
+4. **Sprint Retro + Chat:** Option to share retro takeaways and action items in chat
+5. **Video Conference + Shared Calendar:** Option to add video conference details to events
+6. **Document Sharing + Wiki:** Option to embed public documents in wiki
```

Comment:
Do we have an ADR for document sharing?

Reply:
We do. Here: https://github.com/HHS/simpler-grants-gov/issues/118